common/presenters/events-to-timeline-component.js

```diff
@@ -0,0 +1,66 @@
+const i18n = require('../../config/i18n')
+const addTriggeredEvents = require('../helpers/events/add-triggered-events')
+const setEventDetails = require('../helpers/events/set-event-details')
+
+const getItem = ({
+  component,
+  labelClasses,
+  heading,
+  description,
+  timestamp,
+  byline,
+}) => {
+  return {
+    component,
+    label: {
+      classes: labelClasses,
+      html: heading,
+    },
+    classes: 'app-timeline__item',
+    html: description,
+    datetime: {
+      timestamp,
+      type: 'datetime',
+    },
+    byline: {
+      html: byline,
+    },
+  }
+}
+
+const eventsToTimelineComponent = (moveEvents, move) => {
+  moveEvents = addTriggeredEvents(moveEvents)
+
+  moveEvents = setEventDetails(moveEvents, move)
+
+  const items = moveEvents.map(event => {
+    const { details, event_type: eventType } = event
+    const statusChange = i18n.exists(`events::${eventType}.statusChange`)
+    const labelClasses = statusChange ? 'moj-badge' : undefined
+
+    const heading = i18n.t(`events::${eventType}.heading`, details)
+    const description = i18n
+      .t(`events::${eventType}.description`, details)
+      .replace(/^<br>/, '')
+    // TODO: when we have user info for event, update the following with something like
+    // const byline = i18n.t('events::byline', details)
+    const byline = ''
+
+    const timestamp = event.occurred_at
+
+    // constant for now, but designs allow for variations
+    const component = 'appTimelineItemAlt'
```

Comment:
Can we remove this now we've discussed removing this concept from the component?